From 2642348c276b0d54428239aff0f293022382bc77 Mon Sep 17 00:00:00 2001
From: Aurimas Liutikas <aurimas@google.com>
Date: Tue, 11 Dec 2018 19:14:28 +0000
Subject: [PATCH 2/3] Fix generating id in android namespace

packages/apps/Bluetooth/res/layout/bt_enabling_progress.xml:30: warn: generated id 'android:id/progress' for external package 'android'.

Change-Id: I125e93ff9f9b2f197b62e5b24d80906775d3b5d0
Test: None
---
 res/layout/bt_enabling_progress.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/layout/bt_enabling_progress.xml b/res/layout/bt_enabling_progress.xml
index 7f2fd72b..0e4f870a 100644
--- a/res/layout/bt_enabling_progress.xml
+++ b/res/layout/bt_enabling_progress.xml
@@ -27,7 +27,7 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
-        <ProgressBar android:id="@+android:id/progress"
+        <ProgressBar android:id="@+id/progress"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content" />
 
-- 
2.17.1

